src/rapidfire.cpp

```diff
@@ -32,10 +32,6 @@ Rapidfire::EnableSPIMode()
     digitalWrite(PIN_CLK, LOW);
     digitalWrite(PIN_CS, LOW);
     delay(200);
-    
-    pinMode(PIN_MOSI, INPUT);
-    pinMode(PIN_CLK, INPUT);
-    pinMode(PIN_CS, INPUT);
```

Comment:
I thought we needed to change the pins back to input to prevent drawing excessive current when not in use?